src/pages/Sidebar.js

```diff
@@ -0,0 +1,156 @@
+import {
+  AiFillDatabase,
+  AiFillDelete,
+  AiFillHome,
+  AiFillBank,
+} from 'react-icons/ai';
+import { BiLogOutCircle } from 'react-icons/bi';
+import { HiMenu } from 'react-icons/hi';
+import { NavLink } from 'react-router-dom';
+import { RiCloseLine } from 'react-icons/ri';
+import { getCurrentUser, userLogout } from '../redux/users/usersSlice';
+import '../styles/sidebar.css';
+
+const Sidebar = () => {
+  const currentUser = useSelector((state) => state.user.currentUser);
+  const [navHide, setNavHide] = useState(false);
+  const displayHideNavbar = () => setNavHide(!navHide);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getCurrentUser());
+  }, [dispatch]);
+
+  const handleClick = () => {
+    dispatch(userLogout());
+  };
+
+  return (
+    <>
+      <button
+        type="button"
+        className="close-sidebar sidebar-btn"
+        onClick={displayHideNavbar}
+      >
+        <HiMenu className="text-2xl" />
+      </button>
+      <div
+        className={
+          navHide
+            ? 'sidebar flex h-screen flex-col absolute justify-between border-r bg-white'
+            : 'sidebar-hidden flex h-screen flex-col absolute justify-between border-r'
+        }
+      >
+        <button
+          type="button"
+          className="close-sidebar sidebar-btn"
+          onClick={displayHideNavbar}
+        >
+          <RiCloseLine className="text-2xl" />
+        </button>
+        <div className="px-4 py-6 mt-6">
+          <span className="grid h-20 w-20  place-content-center text-center rounded-full bg-prime text-white">
+            Cherry Rooms
+          </span>
+
+          <nav aria-label="Main Nav" className="mt-6 flex flex-col space-y-1">
+            <NavLink
+              to="/rooms"
+              className="flex items-center gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillHome />
+              </span>
+              <span>Rooms</span>
+            </NavLink>
+
+            <NavLink
+              to="/reservations"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillDatabase />
+              </span>
+              <span>My Reservations</span>
+            </NavLink>
+
+            <NavLink
+              to="/reservation"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillBank />
+              </span>
+              <span>Reserve a Room</span>
+            </NavLink>
+            <NavLink
+              to="/rooms/create-room"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
```

Comment:
This is a repeating class, can make it a constant and use variable instead

SEe where else  in project yo ucan make similar changes, using this as an example.

eg: `const classNav = "flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500";`
`<NabLink className={classNav} ....`